modules/apps/app-builder/app-builder-test/src/testFunctional/tests/AppBuilder.testcase

```diff
@@ -1036,6 +1036,80 @@ definition {
 		AppBuilderAdmin.validateItemNameOnTable(itemName = "Content of form deployed as widget.");
 	}
 
+	@description = "Filter and order the apps created using App Builder"
+	@priority = "5"
+	test FilterAndOrderApps {
+
+		AppBuilderAdmin.addCustomObject(
+			customObjectName = "my custom object",
+			isCreatingFormView = "true",
+			isCreatingViaPlusIcon = "true");
+
+		AppBuilderAdmin.createFormViewAndTableView(
+			fieldType = "Text",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "First App",
+			deployOption = "Product Menu",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "Second App",
+			deployOption = "Standalone",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		App.add(
+			appTitle = "Third App",
+			deployOption = "Widget",
+			formViewName = "My Form View",
+			tableViewName = "My Table View");
+
+		GlobalMenu.gotoPortlet(
+			category = "App Builder",
+			panel = "Control Panel",
+			portlet = "Apps");
+
+		AppBuilderAdmin.validateFilterAndOrder();
+
+		AppBuilderAdmin.chooseAppSearchFilterByOption(filterOption = "Name");
+
+		AppBuilderAdmin.chooseAppSearchFilterByOption(filterOption = "Create Date");
+
+		AppBuilderAdmin.chooseAppSearchFilterByOption(filterOption = "Modified Date");
+
+		AppBuilderAdmin.filterListAscendAndDescend();
+
+		AppBuilderAdmin.chooseMultipleFilter(
+			filterOption = "Deployed",
+			itemName = "Product Menu");
+
+		AppBuilderAdmin.validateListTitleAtRow(
```

Comment:
Instead of using this macro, please, use the `validateItemNameOnTable` and `validateItemNameOnTableNotPresent` macros to make sure that only the filtered App is displayed.